lib/ansible/modules/system/sysctl.py

```diff
@@ -86,6 +86,7 @@
     name: net.ipv4.ip_forward
     value: '1'
     sysctl_set: yes
+    sysctl_file: /dev/null
```

Comment:
We do not believe the above comment means that this is only being set in `/proc` without updating the file, I think the grammar is just poor.

Additionally, we would need an integration test written to verify this behavior, because otherwise I can see this failing right now, or someone breaking this functionality without an explicit test that ensures this functionality.

Reply:
In that case, the comment's wording should be more consistent with the following example, and explicitly mention the sysctl file will be modified as well.

I've tested the behaviour of setting `sysctl_file: /dev/null` with 2.8.5 and it works as expected (only non-persistent change is made and it doesn't fail).

Is there a better way to make non-persistent changes to sysctl with this module?